packages/terra-form-select/src/shared/_Tag.jsx

```diff
@@ -19,17 +20,64 @@ const propTypes = {
    * The value of the tag.
    */
   value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+  /**
+   * Specifies whether the tag is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
 };
 
 /* eslint-disable jsx-a11y/no-static-element-interactions */
-const Tag = ({ children, onDeselect, value }) => {
+const Tag = ({
+  children, onDeselect, value, disabled, intl,
+}) => {
   const theme = React.useContext(ThemeContext);
+  const tagRef = useRef(null);
+
+  const handleKeyPress = (event) => {
+    if ((event.key === 'Enter' || event.key === 'Backspace') && !disabled) {
+      event.stopPropagation();
+      onDeselect(value);
+      const previousLi = tagRef.current.previousElementSibling;
+      const selectInput = tagRef.current.closest('ul').parentElement.parentElement.children[1].children[0];
+      if (previousLi) {
+        const deselectElement = previousLi.children[1];
```

Comment:
why is the index being hard-coded to 1 here ?

Reply:
I have removed the hard coded index. focused input element using `inputRef`.